src/core/middleware/store.ts

```diff
@@ -0,0 +1,45 @@
+import { destroy, invalidator, create } from '../vdom';
+import Store, { StatePaths, Path } from '../../stores/Store';
+import { Process } from '../../stores/process';
+
+const factory = create({ destroy, invalidator });
+
+export const createStoreMiddleware = <S = any>(initial?: Function) => {
```

Comment:
Supports running an optional initial set of processes to bootstrap the store data

Reply:
The factory is used to type the store middleware for the application state